src/coreclr/jit/stacklevelsetter.cpp

```diff
@@ -125,10 +114,21 @@ PhaseStatus StackLevelSetter::DoPhase()
 void StackLevelSetter::ProcessBlock(BasicBlock* block)
 {
     assert(currentStackLevel == 0);
+#ifndef TARGET_X86
```

Comment:
Couldn't we host this and skip the whole process blocks loop in the caller?

Reply:
Ah, good catch. Addressed.